configure.py

```diff
@@ -1228,6 +1228,7 @@ def find_ninja():
                 'service/topology_mutation.cc',
                 'service/topology_coordinator.cc',
```

Comment:
> service: introduce view_building_coordinator
>The coordinator is responsible for building tablet-based views.
>It schedules tasks for `view_building_worker` and updates views'
>statuses.

Please explain about coordination with the load balancer.

Does it prevent view building on two tablet replicas that happen to be on the same node and shard (well, two would be find, but running 100 view builds for 100 tablets is too much).

How does it interact with tablet migrations?

Reply:
Support for tablet operations is added and described a few commits later (174d4b6b5c292a6275a686e02730f73e2e64f6e5).

> Does it prevent view building on two tablet replicas that happen to be on the same node and shard (well, two would be find, but running 100 view builds for 100 tablets is too much).

One shard can be processing only 1 tablet at a time.

Extended this commit message in v9.